Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -307,6 +302,7 @@ def get_analysis_code_reuse_command(intezer_api: IntezerApi, args: dict) -> Comm
                                                 composed_analysis_id=analysis_id,
                                                 sha256='',
                                                 source='',
+                                                extraction_info=None,
```

Comment:
Can you please explain why did you add the extraction_info arg here?

Reply:
This is due to a change in our python SDK that was auto-updated